src/Views/ExpenseForm.elm

```diff
@@ -0,0 +1,55 @@
+module Views.ExpenseForm exposing (view)
+
+import Date
+import Html exposing (Html, text, div, img)
+import Material.Button as Button
+import Material.Options as Options
+import Material.Textfield as Textfield
+import Model exposing (Model, ExpenseType(Debit))
+import Msg exposing (..)
+
+
+isFloat : String -> Bool
+isFloat n =
```

Comment:
I was trying to check it using `Regex` but it is so complicated to a noob like me.